### ISSUE TITLE
Error while put my viewModel to MvvmActivity

### ISSUE THREAD

**Issue by samuelhalim1** [icerockdev/moko-mvvm#77]
i dont know what to do anymore right now..
i already follow the installation step by step..
until this step..

![error](https://user-images.githubusercontent.com/29983404/101077601-a2c51d80-35d7-11eb-816f-1433f5ee0d1b.png)

Help me, please

**Comment by Alex009** [comment]
hi, @samuelhalim1 !
to fix it you should add to your android app module dependencies:
```
implementation("dev.icerock.moko:mvvm:$mokoMvvmVersion")
implementation("androidx.lifecycle:lifecycle-viewmodel-ktx:$lifecycle_version")
implementation("androidx.lifecycle:lifecycle-livedata-ktx:$lifecycle_version")
```

we will fix dependencies configuration later (we will use `api` configuration to transitive dependencies for this case)

**Comment by Alex009** [comment]
also this is not error for compiler. but IDE show error :(

**Comment by Alex009** [comment]
will be released in 0.9.0
